### DISCUSSION TITLE
求求修复下el-select的问题吧

### DISCUSSION THREAD

**Discussion by leoyy123** [element-plus/element-plus#2190] [Q&A]
el-select 的 dropdown 一直都是display:none

**Comment by lvfangren** [comment]
具体是什么问题可以稍微描述下吗？或者有具体的issuse，我很感兴趣这个问题（如果是我理解的那样）

**Reply by yclgkd** [reply]
是说这个组件么
https://element-plus.gitee.io/#/zh-CN/component/select#yuan-cheng-sou-suo
